### PR TITLE
New SelectComponents Script

### PR DESCRIPTION
**Pull request by igorpnk**
A new script has been added that allows you to select components on mechanical layers in single layer mode

### CODE REVIEW

File: Scripts - PCB/SelectComponents/SelectComponentScript.pas [hunk 1]
```diff
@@ -0,0 +1,86 @@
+Procedure StartScript ();
+var
+   Board              : IPCB_Board;         //���������� ������� �������� �����
```
Comment (justinmassiot) [inline Scripts - PCB/SelectComponents/SelectComponentScript.pas]: I don't understand what `Ïåðåìåííàÿ îáúåêòà ïå÷àòíîé ïëàòû` means.
Is there a character encoding issue?

Comment (igorpnk) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: It's simple, this comment is made in Russian. I myself am from Russia and plan to make instructional videos in Russian.

Comment (igorpnk) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: image
Here is how it displays with me

Comment (justinmassiot) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: Okay, so:
1. Let's try to set the encoding of your files to [UTF-8](https://en.wikipedia.org/wiki/Cyrillic_script_in_Unicode), and commit/push again
1. Use english comments and variable/function names as much as possible

Comment (igorpnk) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: How can I do it?
Need to withdraw a request, make changes and create a new request?

Comment (justinmassiot) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: No, please **do not create a new pull request**, just do as I said.

1. Check the encoding of your file, and if necessary, change it to UTF-8.
For example if you use Notepad++ :
change encoding
1. Save you newly encoded file
1. Commit
1. Push

The pull request will automatically update with your latests modifications.

Comment (igorpnk) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: image
I tried changing the encoding. Github in this encoding correctly displays information. But the default Altium designer does not work in this encoding, and incorrectly displays Russian text in the UTF-8 encoding.

Comment (igorpnk) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: Apparently the most correct solution is to translate the comments into English.
I'll do it now.

Comment (igorpnk) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: I have done.

Comment (justinmassiot) [reply Scripts - PCB/SelectComponents/SelectComponentScript.pas]: :+1:

File: Scripts - PCB/README.md [hunk 2]
```diff
@@ -41,6 +41,7 @@ Add-on name & download link | Script description
 [RoomFromPoly](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/Altium-Designer-addons/scripts-libraries/tree/master/Scripts+-+PCB/RoomFromPoly) | Script to create room from selected objects or from selected polygon
 [RotationStep](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/Altium-Designer-addons/scripts-libraries/tree/master/Scripts+-+PCB/RotationStep) | easier way to modify PCB editor rotation step settings (when Spacebar is used to rotate an object)
 [SelectBadConnections](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/Altium-Designer-addons/scripts-libraries/tree/master/Scripts+-+PCB/SelectBadConnections) | This script checks weather Tracks and arcs on signal connect totaly on some other object. Center-to-center check is done. If not, it is selected. Tolerance (and zero tolerance) is supported
+[SelectComponents](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/igorpnk/scripts-libraries/tree/Select_Script/Scripts%20-%20PCB/SelectComponents) | This script allows you to select components in Single Laer Mode to find any layer (Top Overlay, Top Paste and the first 32 mechanical paired layers)
```
Comment (justinmassiot) [inline Scripts - PCB/README.md]:
```suggestion
[SelectComponents](https://minhaskamal.github.io/DownGit/#/home?url=https://github.com/igorpnk/scripts-libraries/tree/Select_Script/Scripts%20-%20PCB/SelectComponents) | This script allows you to select components in Single Layer Mode to find any layer (Top Overlay, Top Paste and the first 32 mechanical paired layers)
```

**Pull request merged**